.php-cs-fixer.dist.php

```diff
@@ -15,6 +15,7 @@
 
 $config = new PhpCsFixer\Config();
 $config->setRiskyAllowed(true)
+    ->setParallelConfig(PhpCsFixer\Runner\Parallel\ParallelConfigFactory::detect())
```

Comment:
is experimental but would improve the performance locally alot: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7777#issuecomment-2134977471 @sulu/core-developer

Reply:
FYI: it's experimental mostly because we wanted to make "canary" adoption, as an opt-in, and to not feel the pressure about potential bugs. But it's well tested and stable enough to just use it 😁.